lib/plugins/filter/before_post_render/backtick_code_block.js

```diff
@@ -63,7 +63,7 @@ function backtickCodeBlock(data) {
       .replace(/{/g, '&#123;')
       .replace(/}/g, '&#125;');
 
-    return `${start}<escape>${content}</escape>${end}`;
+    return `${start}<!--hexoBacktickCodeBlockEscape:${content}:hexoBacktickCodeBlockEscape-->${end}`;
```

Comment:
Let me show you an example:

The post content is:

```markdown
# Title

[code block]
some content
```

Before this PR, those content will escaped to:

```markdown
# Title

<!--\uFFFc1-->
some content
```

So markdown renderer will rendered it into:

```html
<h1>Title</h1>

<!--\uFFFc1-->
<p>some content</p>
```

(Code block escaped as comment will be treated as a blank line, so next line will be wrapped in `<p>`).

But if we simply remove the escaping, there will be no new line before `some content`:

```markdown
# Title

[code block]
some content
```

So it will be rendered into:

```html
<h1>Title</h1>

<!--\uFFFc1-->
some content
```

No `<p>`, which caused unit test to fail.

That's why I bring up this workaround here.

Reply:
I see, it's different from #4161. This is the behavior of the markdown renderer, not Nunjucks.

Reply:
@stevenjoezhang 

However, since https://github.com/hexojs/hexo/pull/4171#issuecomment-594412367 is still there, we have to find out a way to solve this.